CMakeLists.txt

```diff
@@ -827,70 +835,130 @@ install(CODE "
+    file (WRITE \"\$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/lib/python2.7/cgi.py\"
+      \"\${MODIFIED_FILE}\")
+  endif ()
+
+  if(EXISTS \"${CMAKE_INSTALL_PREFIX}/share\" AND IS_DIRECTORY \"${CMAKE_INSTALL_PREFIX}/share\")
+    file(INSTALL FILES ${CMAKE_INSTALL_PREFIX}/share DESTINATION
+      \"\${CMAKE_INSTALL_PREFIX}\"
+         PERMISSIONS USE_SOURCE_PERMISSIONS
+    )
+  endif()
+
+  if(EXISTS \"${CMAKE_INSTALL_PREFIX}/wrappers\" AND IS_DIRECTORY \"${CMAKE_INSTALL_PREFIX}/wrappers\")
+    file(INSTALL FILES ${CMAKE_INSTALL_PREFIX}/wrappers DESTINATION
+      \"\${CMAKE_INSTALL_PREFIX}\"
          PERMISSIONS USE_SOURCE_PERMISSIONS
     )
   endif()
 
   if(EXISTS \"${CMAKE_INSTALL_PREFIX}/bin/man\" AND IS_DIRECTORY \"${CMAKE_INSTALL_PREFIX}/bin/man\")
-    file(INSTALL FILES ${CMAKE_INSTALL_PREFIX}/man DESTINATION ${CPACK_DESTINATION_BIN_PREFIX}
+    file(INSTALL FILES ${CMAKE_INSTALL_PREFIX}/man DESTINATION
+      \"\${CMAKE_INSTALL_PREFIX}/${CPACK_DESTINATION_BIN_PREFIX}\"
          PERMISSIONS USE_SOURCE_PERMISSIONS
     )
   endif()
 
   if(EXISTS \"${CMAKE_INSTALL_PREFIX}/vistrails\" AND IS_DIRECTORY \"${CMAKE_INSTALL_PREFIX}/vistrails\")
-    file(INSTALL FILES ${CMAKE_INSTALL_PREFIX}/vistrails DESTINATION ${CPACK_DESTINATION_PREFIX}
+    file(INSTALL FILES ${CMAKE_INSTALL_PREFIX}/vistrails DESTINATION
+      \"\${CMAKE_INSTALL_PREFIX}\"
          PERMISSIONS USE_SOURCE_PERMISSIONS
     )
   endif()
 
   if(EXISTS \"${CMAKE_INSTALL_PREFIX}/Library\" AND IS_DIRECTORY \"${CMAKE_INSTALL_PREFIX}/Library\")
-    file(INSTALL FILES ${CMAKE_INSTALL_PREFIX}/Library DESTINATION ${CPACK_DESTINATION_PREFIX}
+    file(INSTALL FILES ${CMAKE_INSTALL_PREFIX}/Library DESTINATION
+      \"\${CMAKE_INSTALL_PREFIX}\"
          PERMISSIONS USE_SOURCE_PERMISSIONS
     )
   endif()
+  
+  file(INSTALL FILES ${cdat_BINARY_DIR}/build_info.txt DESTINATION ${CMAKE_INSTALL_PREFIX}/info)
 
   # Unset QT_LIB_DIR as we need to use the one in user's environment
   # We need to keep in ming that we might need to build Qt on some systems
   # (e.g. RH6) in which case this might break something
   set(QT_LIB_DIR)
+
+  # Configure the environment setup script to point to the installation
+  # Creating a temporary file that will be installed.
   configure_file(${cdat_CMAKE_SOURCE_DIR}/cdat_modules_extra/setup_runtime.sh.in
-    ${CPACK_DESTINATION_BIN_PREFIX}/setup_runtime.sh
+    \"\$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CPACK_DESTINATION_BIN_PREFIX}/setup_runtime.sh\"
     @ONLY
   )
 
+  # Finally, create a symlink for python to point to point to installed python
+  if (EXISTS \"\$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CPACK_DESTINATION_BIN_PREFIX}/python2.7\"
+      AND
+      NOT EXISTS \"\$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CPACK_DESTINATION_BIN_PREFIX}/python\")
+    execute_process(COMMAND \${CMAKE_COMMAND} -E create_symlink
+      \"\$ENV{DESTDIR}/\${CMAKE_INSTALL_PREFIX}/${CPACK_DESTINATION_BIN_PREFIX}/python2.7\"
```

Comment:
Probably a merge artifact; I think I was poking stuff with a stick to make it work (and added the above configure_file) and you did this part. We can remove my configure_file stuff.